AK/String.h

```diff
@@ -51,6 +51,10 @@ class String : public Detail::StringBase {
     // Creates a new String from a sequence of UTF-8 encoded code points.
     static ErrorOr<String> from_utf8(StringView);
 
+    // Creates a new String using the replacement character for invalid bytes
+    [[nodiscard]] static String from_bytes_with_replacement(ReadonlyBytes);
+    [[nodiscard]] static String from_bytes_with_replacement(StringView);
```

Comment:
Hmm, how about `from_utf8_with_replacement_character` just to make it ultra explicit?

Reply:
Sounds good!

Reply:
Although, the `from_utf` is potentially a bit confusing since it can take something that isn't valid utf8

Reply:
Right, but isn't the idea that it does expect UTF-8, but handles invalid characters by substitution rather than failure?

Reply:
Right! Yeah I thought about it some more and indeed that seems much more clear :^)